source/locale/zh_CN/LC_MESSAGES/reference/distributions.po

```diff
@@ -31,18 +31,18 @@ msgid ""
 "specific parameters, such as the ``low`` and ``high`` endpoints for a "
 ":class:`~optuna.distributions.UniformDistribution`."
 msgstr ""
-":mod:`~optuna.distributions` 模块定义了代表各种概率分布的类，"
-"主要用于优化 trial 时进行初始超参数建议。"
+":mod:`~optuna.distributions` 模块定义了代表各种概率分布的类, "
+"主要用于优化 trial 时进行初始超参数建议."
```

Comment:
If we follow the [standard](http://www.moe.gov.cn/ewebeditor/uploadfile/2015/01/13/20150113092346124.pdf) strictly, yes `。` is correct. But still there're lot's of unsettled debates (e.g. [中英文混排的时候如何使用标点符号？](https://www.zhihu.com/question/19695720)) on this topic so any system that is consistent across all texts, I think, should be fine. Here we use `.` and `.` rather than `，` and `。` because halfwidth punctuation system, compared with fullwidth system is simpler and more friendly to both the translator and sphinx-intl pulgin.

Reply:
From the zhihu link you sent, it seems people all agree that using `。` is correct for Chinese. But since using the English punctuation is more friendly to both the translator and sphinx-intl plugin, then I think it is OK. Will you change all the `。` and `，` in all the translation?

Reply:
Yes this will be updated in the future as it involves many documents so replacing them now will cause chaos in our PR